tools/browser-matcher/common-matchers.mjs

```diff
@@ -22,3 +22,14 @@ export const supportsFetch = new SpecMatcher()
   .include('firefox>=40')
   .include('ios>=11')
   .include('android')
+
+// Some browsers support basic fetch API, but not all supporting functions.
+// E.g. arrayBuffer on ios@10 generates an error when used with FormData instance.
+// MDN shows this function as not supported: https://developer.mozilla.org/en-US/docs/Web/API/Body/arrayBuffer
+export const supportsFetchExtended = new SpecMatcher()
+  .include('safari>=11.1') // MDN says no support (11.1 currently latest), but 11.1 is accounted for in the tests
+  .include('chrome>=42')
+  .include('edge>=14')
+  .include('firefox>=40')
+  .include('ios>=11.3')
+  .include('android')
```

Comment:
Unsure of the relevance and history of the `fetchExt` (supportsFetchExtended) matcher. Seems like it's probably not needed given how old the browsers are (ios>=11.3, safari>=11.1), and we could just use the supportsFetch matcher now.

Reply:
I think this depends on how far back we want to support the "all browsers" testing.

Reply:
The same question of relevance applies to the old versions in our supportsFetch matcher, I suppose. I'll leave it for now for clarity around potential future old browser edge cases.